compute/ping_test.go

```diff
@@ -74,37 +72,11 @@ func TestPing(t *testing.T) {
 			t.Error("expected pingOut to be nil")
 		}
 
-		if err.Error() != blankError {
+		if !strings.Contains(err.Error(), "unable to ping") {
```

Comment:
Is this right?  Is there a different helper method we should be using instead?  This seems like a grey area, so I have no idea.

Reply:
Not for this specific error type - this is something we will wrap in future i'd say